0MI0800256/task02.07.03.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+int main()
+{
+    ifstream file ("MyFile.txt");
+
+    if(!file.is_open())
+    {
+        cout << "Error!" << endl;
+        return -1;
+    }
+    int count = 0;
+    while(!file.eof())
```

Comment:
It would've been nice to create a function that get's invoked inside the `main` :P